fetch/api/cors/cors-expose-star.js

```diff
@@ -0,0 +1,31 @@
+if (this.document === undefined) {
+  importScripts("/resources/testharness.js");
+  importScripts("../resources/utils.js");
+}
+
+const url = "http://{{host}}:{{ports[http][1]}}" + dirname(location.pathname) + RESOURCES_DIR + "top.txt",
+      sharedHeaders = "?pipe=header(Access-Control-Expose-Headers,*)|header(Test,X)|header(Set-Cookie,X)|"
+
+promise_test(t => {
```

Comment:
No need to fix, but my personal preference is to exclude arguments that aren't used, so that one doesn't have to look for where they're used and fail, as I just did.